test/index.js

```diff
@@ -57,30 +60,30 @@ describe('require("pom-parser")', function () {
   describe('when opts is null', function(){
     it('parser should throw an error', function() {
       assert.throws(function(){
-        pomParser.parse(null, function(err, response){}, Error);
+        parse(null, function(err, response){}, Error);
       });
     });
   });
 
   describe('when opts is empty', function(){
     it('parser should throw an error', function(){
-      assert.throws(function(){ 
-        pomParser.parse({}, function(err, response){}, Error);
+      assert.throws(function(){
+        parse(null, function(err, response){}, Error);
```

Comment:
Is this change of changing argument from `{}` to `null` required?

Reply:
i think it was a typo, removed it.